SMLHelper/Patchers/PrefabDatabasePatcher.cs

```diff
@@ -14,33 +14,33 @@
 
     internal static class PrefabDatabasePatcher
     {
-        private static class PostPatches
-        {
-            [PatchUtils.Postfix]
-            [HarmonyPatch(typeof(PrefabDatabase), nameof(PrefabDatabase.LoadPrefabDatabase))]
-            internal static void LoadPrefabDatabase_Postfix()
+            private static class PostPatches
```

Comment:
Revert all of the changes to this file

Reply:
I'm slightly confused, because on my end, it doesn't appear I changed anything.

Reply:
You seem to have inadvertantly added a single indentation to every line inside PrefabDatabasePatcher. Likely your IDE is showing them as the same, as the only change is whitespace, but they are not.